content/strategy-goals/strategy/code-graph/code-intelligence/index.md

```diff
@@ -79,55 +79,54 @@ Recent key learnings:
 
 ## Strategy and Plans
 
-### FY22 Q4 goals
-
-These tie back to [FY22 Q4 Product/Engineering OKRs](../../../goals/2022_Q4.md#product-engineering)
+### Themes
 
-**Objective: Make the power of our features easier to find and use**
+**Unlock dependency navigation**
 
-- **KR:** Measure and increase WAU for customer facing product teams by 15%. For most teams this will require understanding and improving time-to-value as a driver for increasing active users.
+We believe this is Code Intelligence's killer feature. Most developers think in terms of package and dependency versions rather than focusing on external repositories. Supporting precise dependency navigation elevates the code navigation experience to a new level of cross-project analysis. The ability to navigate to any third party dependency a repository references is key to use cases like [Developer onboarding](../../use-cases/dev-onboarding.md) where time to value is directly related to how fast a developer is able to navigate and understand a new codebase (including the dependencies it relies on).
```

Comment:
I might say here that it is emerging as an additional killer feature. Existing code intelligence is also a killer feature. :)

Reply:
This section is a really great addition and really explains why this is important. Maybe worth linking to Olaf's video? Is that internal only?

Reply:
Nice nitpick, I'll add it :)

Reply:
Yeah you can either restrict it and mark it as (internal only), or plan to record another public video later. Both are fine.